test/unit/table_emission_test.sql

```diff
@@ -709,5 +709,22 @@ select results_eq(
   'Vented emissions with an emission_type in swrs.taxed_venting_emission_type are populated with the correct fuel_mapping_id'
 );
 
+select results_eq(
```

Comment:
Does this cover everything? I looked at adding the `ar5_calculated_quantity` to the above tests but `swrs_transform.emission` didn't have that column